CONTRIBUTING.rst

```diff
@@ -19,3 +19,16 @@ was number 123. See the github documentation for more information on this
 topic: https://help.github.com/articles/closing-issues-using-keywords/
 
 If you need help figuring out how to do this, please comment on your PR! We can help or do it for you.
+
+When developing your tests, you can disable cleanup functions so any machines
+that are turned on by scans stay on and all objects created on server persist.
+This can be useful when running individual tests, but is not advisable for
+running the whole test suite. To run a specific test with cleanup functionality
+disabled, run::
+
+    #Run a specific test with cleanup disabled
+    DEBUG=True py.test camayoc/tests/qpc/api/v1/authentication/test_login.py
```

Comment:
I have some concerns about the debug flag:

* It will leave vCenter machines on and will require manual work to get them off and free up resource usage.
* It won't clear entities which may act as side effect for other tests. In addition to that on our automation job the slave is a throw away one so we won't be able to inspect the system there. If that is aiming local run than I would recommend using PDB.